### PR TITLE
UIFR-214: Allow including CSS and JS files not in scripts and styles …

### PR DESCRIPTION
**Pull request by mogoodrich**
…subdirectory

### CODE REVIEW

File: api/src/main/java/org/openmrs/ui/framework/UiUtils.java [hunk 1]
```diff
@@ -30,68 +21,81 @@
...
+        includeCss(null, file, null);
+    }
+
+    public void includeCss(String providerName, String file) {
+        includeCss(providerName, file, null);
+    }
+
+    public void includeCss(String providerName, String file, Integer priority) { includeCss(providerName, file, priority, false);}
+
+    public void includeCss(String providerName, String file, Integer priority, Boolean excludeStylesPrefixFromPath) {
```
Comment (mogoodrich) [inline api/src/main/java/org/openmrs/ui/framework/UiUtils.java:66]: I'm not crazy about the way I chose to do this (adding an "excludeBy..." flag, but I couldnt' come up with a better way.

Comment (brandones) [reply api/src/main/java/org/openmrs/ui/framework/UiUtils.java:66]: `pathIsRelativeToStyles`, defaulting to true?

Comment (brandones) [reply api/src/main/java/org/openmrs/ui/framework/UiUtils.java:66]: Also, any reason not to use the primitive `boolean`?

Comment (mogoodrich) [reply api/src/main/java/org/openmrs/ui/framework/UiUtils.java]: I like that better, will switch, thanks Brandon!

**Pull request merged**